pkgs/top-level/all-packages.nix

```diff
@@ -10138,6 +10138,9 @@ with pkgs;
 
   importNpmLock = callPackages ../build-support/node/import-npm-lock { };
 
+  inherit (callPackages ../build-support/node/fetch-pnpm-deps { })
+    fetchPnpmDeps_7_0 fetchPnpmDeps_6_0;
```

Comment:
I think setting a default might cause breakages down the line if we ever need to drop an older version (like when it has a security vulnerability).

I have used 6_0 for now, as all packages use the 6.0 lock file format. We can probably use 7_0 too, but I thought this has the least likelihood of breakages

Reply:
> I think setting a default might cause breakages down the line if we ever need to drop an older version (like when it has a security vulnerability).
> 
> I have used 6_0 for now, as all packages use the 6.0 lock file format. We can probably use 7_0 too, but I thought this has the least likelihood of breakages

OK I see. It'd be nice perhaps to explain in a comment that we don't consider any lock file format as the "default" or "standard" or "most common" format, hence we don't set a default.

Reply:
I have changed the function names again. Instead of reflecting the latest supported lockfile version, it's not just a number that will be incremented when needed. We can't use `pnpm 9.0.6` for projects that define `8.x.x` in their `package.json`